docker-compose-demo.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  demo-data:
+    image: mekomsolutions/ozone-demo-data
+    environment:
+      - SPRING_PROFILES_ACTIVE=prod
+      - OPENMRS_SERVERURL=${SERVER_SCHEME}://${O3_HOSTNAME}/openmrs
```

Comment:
Can we cross check if the variable is `OPENMRS_SERVERURL` or `OPENMRS_SERVER_URL`